content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -90,11 +103,18 @@ algorithmSource:
 ...
 
 disablePreemption: true
-
 ```
 
-Although preemption of the scheduler is enabled by default, it is disabled if `PodPriority`
-feature is disabled.
+### Start an older version of Rescheduler in the cluster
+
+When priority or preemption is disabled, we must start Rescheduler v0.3.1
+(instead of v0.4.0) to ensure that critical Pods are scheduled when nodes or
+cluster are under resource pressure. Since we do not plan to remove critical
+annotation of Pods, running Rescheduler should be enough and no other changes to
+the configuration of Pods should be needed.
+
+Changing the rescheduler back to v.0.3.1 is the reverse of
```

Comment:
I think most people would use container images. We should point them to image location of rescheduler.

Reply:
good point. done.